ClothesApp/Infrastructure/Repositories/AddressRepository.cs

```diff
@@ -0,0 +1,57 @@
+using System.Linq.Expressions;
+using Application.Interfaces.Repositories;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Repositories;
+
+public class AddressRepository : IAddressRepository
+{
+    private readonly ClothesAppContext _dbContext;
+
+    public AddressRepository(ClothesAppContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public Task<Address> Insert(Address entity)
+    {
+        throw new NotImplementedException();
```

Comment:
So how does your application work without this implementation?

Reply:
Yes, because there is no endpoint to add an address. I'll add it